packages/graphql/tests/integration/sort.int.test.ts

```diff
@@ -243,108 +243,136 @@ describe("sort", () => {
             await session.close();
         });
 
-        test("should sort on top level", async () => {
-            await Promise.all(
-                ["ASC", "DESC"].map(async (type) => {
-                    const query = gql`
-                        query($actorNames: [String!]!, $direction: SortDirection!) {
-                            actors(
-                                where: { name_IN: $actorNames }
-                                options: { sort: [{ totalScreenTime: $direction }] }
-                            ) {
-                                name
-                                totalScreenTime
-                            }
-                        }
-                    `;
+        test("should sort DESC on top level", async () => {
```

Comment:
Any reason these tests have been split? 🧐

Reply:
Oh, good catch, forgot to mention in the description.

Generally, dynamic flows in tests are considered a bad practice (with some exceptions) as that increases the complexity of the test, and thus, the chances of it having bugs (as you can see, the disabled rule `jest/no-conditional-expect` exists because of this).